assets/dev/js/editor/editor-base.js

```diff
@@ -786,13 +786,13 @@ export default class EditorBase extends Marionette.Application {
 
 		this.addDeprecatedConfigProperties();
 
+		elementorCommon.elements.$window.trigger( 'elementor:loaded' );
```

Comment:
Consider swaping of 'elementor:loaded' and 'elementor:init' events

Reply:
Added as a task.